provider/lxd/testing_test.go

```diff
@@ -503,7 +503,9 @@ func (conn *StubClient) RemoveInstances(prefix string, ids ...string) error {
 	return nil
 }
 
-func (conn *StubClient) FindImage(series, arch string, sources []lxd.RemoteServer) (lxd.SourcedImage, error) {
+func (conn *StubClient) FindImage(
+	series, arch string, sources []lxd.RemoteServer, copyLocal bool, callback environs.StatusCallbackFunc,
+) (lxd.SourcedImage, error) {
 	conn.AddCall("EnsureImageExists", series, arch)
```

Comment:
this seems strange that FindImage records "EnsureImageExists" has been called.
I probably missed it the first time around (other than noting StubClient should go away entirely in favor of a mockgen version)